modules/cnf-programming-the-n3000-operator.adoc

```diff
@@ -0,0 +1,351 @@
+----
+
+. Verify that the bitstream ID of the card has changed:
++
+[source,terminal]
+----
+oc get n3000node node1 -o yaml
+----
++
+.Example output
+[source,yaml]
+----
+status:
+      conditions:
+          - lastTransitionTime: "2020-12-15T18:18:53Z"
+            message: Flashed successfully <1>
+            observedGeneration: 2
+            reason: Succeeded
+            status: "True"
+            type: Flashed
+      fortville:
+      - N3000PCI: 0000:1b:00.0
+        NICs:
+        - MAC: 64:4c:36:11:1b:a8
+          NVMVersion: 7.00 0x800052b0 0.0.0
+          PCIAddr: 0000:1a:00.0
+          name: Ethernet Controller XXV710 Intel(R) FPGA Programmable Acceleration Card N3000 for Networking
+        - MAC: 64:4c:36:11:1b:a9
+          NVMVersion: 7.00 0x800052b0 0.0.0
+          PCIAddr: 0000:1a:00.1
+          name: Ethernet Controller XXV710 Intel(R) FPGA Programmable Acceleration Card N3000 for Networking
+        - MAC: 64:4c:36:11:1b:ac
+          NVMVersion: 7.00 0x800052b0 0.0.0
+          PCIAddr: 0000:1c:00.0
+          name: Ethernet Controller XXV710 Intel(R) FPGA Programmable Acceleration Card N3000 for Networking
+        - MAC: 64:4c:36:11:1b:ad
+          NVMVersion: 7.00 0x800052b0 0.0.0
+          PCIAddr: 0000:1c:00.1
+          name: Ethernet Controller XXV710 Intel(R) FPGA Programmable Acceleration Card N3000 for Networking
+      fpga:
+      - PCIAddr: 0000:1b:00.0 <2>
+        bitstreamId: "0x2315842A010601" <3>
+        bitstreamVersion: 0.2.3
+        deviceId: "0x0b30 <4>
+----
++
+<1> The message field indicates the device is successfully flashed.
+<2> Use this field to check that the correct PCI address of the card is programmed.
+<3> Use this field to verify that the `bitstreamId` is changed. Check this against the output obtained from step 5.
+<4> The current deviceID of the bitstream inside the card exposed to the system.
+
+. Check the FEC PCI devices on the node:
+
+.. Ensure the node configuration is applied correctly:
++
+[source,terminal]
+----
+$ oc debug node/node1
+----
+.Expected output
++
+[source,terminal]
+----
+Starting pod/<node-name>-debug ...
+To use host binaries, run `chroot /host`
+
+Pod IP: <ip-address>
+If you don't see a command prompt, try pressing enter.
+
+sh-4.4#
+----
+.. Verify that you can use the node file system:
++
+[source,terminal]
+----
+sh-4.4# chroot /host
+----
+.Expected output
++
+[source,terminal]
+----
+sh-4.4#
+----
+
+.. List the PCI devices associated with the accelerator on your system:
++
+[source,terminal]
+----
+$ lspci | grep accelerators
+----
+.Expected output
++
+[source,terminal]
+----
+1b:00.0 Processing accelerators: Intel Corporation Device 0b30
+1d:00.0 Processing accelerators: Intel Corporation Device 0d8f (rev 01)
+
+----
+Devices belonging to the FPGA are reported in the output, where Device ID `0b30` is the RSU interface used to program the card, and the `0d8f` is a physical function of the newly programmed 5G device.
```

Comment:
Add a blank line above, use + if you want to keep the text with the step.  Omit the + if this paragraph is a summary of the verification.

My sugg is to replace the first comma with a period and make two sentences.  You can remove "where".